File: codegen/codegen-erlang/src/test/resources/ir/encode_query_value_rest_json.expected.erl
encode_query_value(V) when is_boolean(V) -> atom_to_binary(V, utf8);
encode_query_value(V) when is_integer(V) -> integer_to_binary(V);
encode_query_value(V) when is_float(V) -> float_to_binary(V);
encode_query_value(V) when is_binary(V) -> V;
encode_query_value(V) when is_atom(V) -> atom_to_binary(V, utf8).

File: codegen/codegen-elixir/src/test/resources/ir/encode_query_value_rest_json.expected.ex
defp encode_query_value(v) when is_boolean(v), do: Atom.to_string(v)
defp encode_query_value(v) when is_integer(v), do: Integer.to_string(v)
defp encode_query_value(v) when is_float(v), do: Float.to_string(v)
defp encode_query_value(v) when is_binary(v), do: v
defp encode_query_value(v) when is_atom(v), do: Atom.to_string(v)